src/main/webapp/app/shared/constants/file-extensions.constants.ts

```diff
@@ -0,0 +1 @@
+export const FILE_EXTENSIONS = '.png, .jpg, .jpeg, .svg, .pdf, .zip, .doc, .docx, .csv, .xls, .xlsx, .ppt, .pptx, .pages, .rtf, .numbers, .key, .odt, .ods, .odp, .odg, .odf.';
```

Comment:
Why do you need this as well as the definition in file-uploader.service.ts? This one is only used for showing the extension to the user and could be generated from the other one or both could be generated from an array if that makes it easier.

Reply:
Also, this String does not mention `pages-tef` as an extension, while the other one does. And the order of the file extensions is different between the two. This is exactly the reason why you should remove this duplication.

Reply:
I agree, the inconsistency and duplication should be fixed

Reply:
Fixed the inconsistency by using an array for both code places.